context.tf

```diff
@@ -19,7 +19,7 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.24.1"
```

Comment:
@martijnvdp updates to context.tf need to be directly copied from https://github.com/cloudposse/terraform-null-label/blob/master/exports/context.tf. Bumping this version number is not enough. Mind copying that `context.tf` to here and in `examples/complete/`?

Reply:
thanks i did not know that i will do so